roles/openshift_ca/tasks/main.yml

```diff
@@ -19,7 +19,8 @@
 
 - name: Reload generated facts
   openshift_facts:
-  when: hostvars[openshift_ca_host].install_result is changed
+  when:
+  - hostvars[openshift_ca_host].install_result | default({'changed':false}) | changed
```

Comment:
Slight syntax change on the filters.  `| changed` should be `is changed`.  See https://github.com/openshift/openshift-ansible/pull/6484